README.md

```diff
@@ -76,9 +75,10 @@ Only one of MUST_GATHER_SINCE_TIME / MUST_GATHER_SINCE may be used
 oc adm must-gather --image=quay.io/modh/must-gather:rhoai-2.17 --since=3h
 ```
 
-If you have enabled customized namespaces for installtion, below env. variable need to be configed before running "oc adm must-gather":
-export OPERATOR_NS=<your-operator-namespace>
-export APPLICATIONS_NS=<your-applications-namespace>
+If you have enabled customized namespaces for installtion, below env. variable need to be configed when running "oc adm must-gather", example:
+```
+oc adm must-gather --image=quay.io/modh/must-gather:rhoai-2.17 -- "export OPERATOR_NAMESPACE=<your-operator-namespace>;export APPLICATIONS_NAMESPACE=<your-application-namespace>; /usr/bin/gather"
```

Comment:
Will just "OPERATOR_NAMESPACE=<your-operator-namespace>  APPLICATIONS_NAMESPACE=<your-application-namespace> /usr/bin/gather"  (per process environment instead of export)? Just to make it shorter.

Reply:
as long as test shows it is working, we can update readme.